jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+	<title>API - JWQL</title>
+
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="container">
+        <h2>Explore the JWQL API</h2><hr>
+        <p align="justify">
+            <ul>
+                <li><a href="#introduction">What is the JWQL API?</a></li>
+                <li><a href="#examples">Examples</a></li>
+                <li><a href="#api_list">List of Avaliable Services</a></li>
+                <li><a href="https://jwql.readthedocs.io/en/latest/index.html?highlight=api">Documentation</a></li>
+            </ul>
+        </p>
+        <br>
+        
+        <div id="introduction">
+            <h4>What is the JWQL API?</h4><hr>
+            <p align="justify">
+                JWQL allows users to access the data available through our programmatic Application Programming Interface (API). In short, an API
+                accepts requests for services from a system, gathers the items in the request, and returns a response to the user. A popular example used to describe the
+                functionality of an API is describing the role of a waiter in a restaurant. When you enter a restaurant and sit at your table, you are typically
+                provided a menu containing all of the drink and food options. But how do you communicate your food and drink order to
+                the kitchen staff? This is where the waiter (API) comes in, you provide the waiter with your order (request), they will communicate
+                your order to the kitchen staff (system), and bring your food and drinks when they are prepared (the response). The waiter (API) disguises
+                all of the steps that are necessary to getting your final product delivered (cooking the food, counting stock, knowing recipes, etc.) but, is the 
+                link to the user and the response they wish to obtain from a system.
+            </p>
+        </div>
+        <br>
+
+        <div id="examples">
+            <h4>Examples</h4><hr>
+            <p align="justify">
+                Here we showcase a few examples of a few of the APIs that we offer. To see a full list of supported APIs click <a href="#api_list">here</a>.
+            </p><br>
+            <h5>Obtaining Filenames Associated with a Rootname</h5>
+            <p align="justify">
+                A user is interested in all of the filenames associated with a specific JWST rootname.
+            </p>
+            <pre>
+                <code class="python">
+                    from urllib import request
+                    import json
+
+                    # Build the URL https://dljwql.stsci.edu/api/roontname/service/
+                    url = 'https://dljwql.stsci.edu/api/jw93025001001_02102_00001_nrca2/filenames/' 
+                    r = request.urlopen(url) 
```

Comment:
This sounds kind of stupid but I would prefer that this variable be changed from `r` to something like `req` or `data`, since PEP8 discourages single-letter variable names.

Reply:
Good idea, I will make the change!